### PR TITLE
Add ANSI with Arrow Layout & RGB Underglow as a Caps Lock Indicator

### PR DESCRIPTION
**Pull request by mrsendyyk**
## Description

1.  Add ANSI layout with arrow
2.  Add RGB Underglow as a caps lock indicator

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mrsendyyk** [comment]
Hi @fauxpark, can you help me review to this pull request?
This is my first contribution.

File: keyboards/dz60/keymaps/mrsendyyk/keymap.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+// RGB Underglow as a Caps Lock Indicator
+
+void led_set_user(uint8_t usb_led) {
+    if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
+        DDRB |= (1 << 2); PORTB &= ~(1 << 2);
+        rgblight_setrgb(100, 255, 100);
+    } else {
+        DDRB &= ~(1 << 2); PORTB &= ~(1 << 2);
+        rgblight_setrgb(0, 0, 0);
+    }
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/mrsendyyk/keymap.c:62]:
```suggestion
    if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
        writePinLow(B2);
        rgblight_setrgb(100, 255, 100);
    } else {
        rgblight_setrgb(0, 0, 0);
    }
```

Comment (mrsendyyk) [reply keyboards/dz60/keymaps/mrsendyyk/keymap.c]: Done

Comment (mrsendyyk) [reply keyboards/dz60/keymaps/mrsendyyk/keymap.c]: Whoa sorry, i'm accidentally removed request for qmk/collaborators.

**Comment by mrsendyyk** [comment]
Whoa sorry, i'm accidentally removed request for qmk/collaborators.

**Comment by mrsendyyk** [comment]
Hi @fauxpark, can you requested a review from qmk/collaborators again? I'm accidentally removed request for qmk/collaborators.
Thank you

**Comment by noroadsleft** [comment]
Thanks!

**Comment by mrsendyyk** [comment]
> 
> 
> Thanks!

Thank you @fauxpark & @noroadsleft

**Pull request merged**